src/components/EntityInfo/EntityMenu.vue

```diff
@@ -367,7 +383,7 @@ export default class EntityMenu extends Mixins(AllowableActionsMixin) {
 
   /** The tooltip text for AGM Extension list item. Text is different if action item is disabled. */
   get agmExtensionToolTipText (): string {
-    if (!this.isAllowed(AllowableActions.AGM_EXTENSION)) {
+    if (!this.isAllowed(AllowableActions.AGM_EXTENSION) && !this.IsAuthorized(AuthorizedActions.AGM_EXTENSION_FILING)) {
```

Comment:
You won't need this if the action is hidden when not authorized.

Ditto below.

Reply:
Obsolete. Resolving.